src/app/contexts/ToggleContext/utils/constructTogglesEndpoint/index.js

```diff
@@ -1,13 +1,6 @@
-import onClient from '#lib/utilities/onClient';
-
 export default (service, origin) => {
   const requestOrigin = origin || 'https://www.test.bbc.com';
   const baseTogglesUrl = `${process.env.SIMORGH_TOGGLES_URL}/toggles?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
 
-  // client side renders should trigger a geoIPLookup
-  if (onClient()) {
-    return `${baseTogglesUrl}&geoiplookup=true`;
```

Comment:
This removes the onClient logic, since now that we are not carrying out a GeoIP lookup via the Toggles API, we do not need to fetch data from Toggles on the client. We can do this on the server.